[CON-1576] Create Integrations > SentinelOne doc (#1894)
SumoLogic/sumologic-documentation: docs/integrations/saas-cloud/sentinelone.md

---
id: sentinelone
title: Sumo Logic App for SentinelOne
sidebar_label: SentinelOne

> description: The Sumo Logic App for the SentinelOne App offers functionality for monitoring and analyzing your organization's SentinelOne Threats and Agents logs.

Author: Resolved

---

import useBaseUrl from '@docusaurus/useBaseUrl';

<img src={useBaseUrl('img/send-data/sentinelone-icon.png')} alt="sentinelone-icon.png" width="100" />

The SentinelOne App for Sumo Logic is designed to provide security professionals with a comprehensive view of their organization's security posture. Based on Sumo Logic’s SentinelOne Source, this app allows you to quickly ingest data from your SentinelOne agents into Sumo Logic for real-time analysis.

With the SentinelOne App, you can gain valuable insights into your endpoint security data, including threat intelligence, endpoint activity, and system health. You can monitor your endpoints for suspicious behaviour and quickly respond to any security incidents that may arise. 
The app provides pre-built dashboards that enable you to quickly visualize your SentinelOne data and gain insights into your organization's security posture. You can customise these dashboards to meet your specific needs and monitor the metrics that matter most.

> With the SentinelOne App for Sumo Logic, you can:

Author: Resolved

* Ingest data from SentinelOne agents into Sumo Logic for real-time analysis and response
* Monitor your endpoints for suspicious behaviour and respond to security incidents in real-time
* Gain valuable insights into your organization's security posture with pre-built dashboards
* Customize dashboards to meet your specific needs and monitor the metrics that matter most to you

In summary, the SentinelOne App for Sumo Logic provides security professionals with the tools they need to monitor and respond to security incidents in real-time, ensuring the protection of their organization's critical assets.

## Log Types

The Sumo Logic App for SentinelOne consumes Threats and Agents logs, refer to the [Threats](https://usea1-partners.sentinelone.net/api-doc/api-details?category=threats&api=get-threats) and [Agents](https://usea1-partners.sentinelone.net/api-doc/api-details?category=agents&api=get-agents) documentation.

### Sample Log Messages

> **Sample Threats Log Messages**
> 
> ```json

Reviewer: ```suggestion
```json title="Sample Threats Log Messages"
```

Author: Resolved

        "fileVerificationType": null,
        "identifiedAt": "2023-03-16T18:23:01Z",
        "incidentStatus": "resolved",
        "incidentStatusDescription": "Resolved",
        "initiatedBy": "agent_policy",
        "initiatedByDescription": "Agent Policy",
        "initiatingUserId": null,
        "initiatingUsername": null,
        "isFileless": false,
        "isValidCertificate": true,
        "maliciousProcessArguments": null,
        "md5": null,
        "mitigatedPreemptively": false,
        "mitigationStatus": "marked_as_benign",
        "mitigationStatusDescription": "Marked as benign",
        "originatorProcess": "launchd",
        "pendingActions": false,
        "processUser": "root",
        "publisherName": "JAMF Software",
        "reachedEventsLimit": null,
        "rebootRequired": false,
        "sha1": "7c5d6774855fb32785f73ec2095",
        "sha256": null,
        "storyline": "5D3A-284D-46-A779-1072FA4",
        "threatId": "164213801585",
        "threatName": "jamf",
        "updatedAt": "2023-03-16T18:30:37.284743Z"
    },
    "whiteningOptions": [
        "certificate",
        "hash",
        "path"
    ]
}
```

### Sample Query for Threats

> ```sql="Suspicious threats detected"

Reviewer: ```suggestion
```sql title="Suspicious threats detected"
```

Author: Resolved

_sourceCategory=sentinelone  threatInfo
| json "id", "threatInfo.incidentStatus", "threatInfo.classificationSource", "threatInfo.confidenceLevel", "threatInfo.detectionEngines", "threatInfo.initiatedBy", "threatInfo.rebootRequired", "threatInfo.createdAt", "threatInfo.detectionType", "threatInfo.mitigationStatus", "threatInfo.analystVerdict", "threatInfo.threatName", "agentRealtimeInfo.siteName","agentRealtimeInfo.agentOsType", "agentRealtimeInfo.agentMachineType" as id, incidentStatus, classificationSource, confidenceLevel, detectionEngines, initiatedBy, rebootRequired, createdAt, detectionType, mitigationStatus, analystVerdict, threatName, siteName, agentOsType, agentMachineType nodrop
| where siteName matches "{{site}}" or isNull(siteName)
| where mitigationStatus matches "{{mitigation_status}}" or isNull(mitigationStatus)
| where agentOsType matches "{{os}}" or isNull(agentOsType)
| where agentMachineType matches "{{machine}}" or isNull(agentMachineType)
| where confidenceLevel matches "{{confidence_level}}" or isNull(confidenceLevel)
| where confidenceLevel = "suspicious"
| count_distinct(id)
```

**Sample Agents Log Messages**

> ```json

Author: Resolved

            ],
            "name": "Wi-Fi",
            "physical": "q2:5d:a0:2d:0d:95"
        }
    ],
    "networkQuarantineEnabled": false,
    "networkStatus": "connected",
    "operationalState": "na",
    "operationalStateExpiration": null,
    "osArch": "64 bit",
    "osName": "Windows 10 Pro",
    "osRevision": "19044",
    "osStartTime": "2023-03-17T10:22:20Z",
    "osType": "windows",
    "osUsername": null,
    "rangerStatus": "Enabled",
    "rangerVersion": "12.11.1.16",
    "registeredAt": "2022-08-09T19:28:45.932112Z",
    "remoteProfilingState": "disabled",
    "remoteProfilingStateExpiration": null,
    "scanAbortedAt": null,
    "scanFinishedAt": "2022-08-10T08:02:01.732412Z",
    "scanStartedAt": "2022-08-09T19:31:28.742739Z",
    "scanStatus": "finished",
    "serialNumber": "FS6V1B3",
    "siteId": "1138463255859336",
    "siteName": "Default site",
    "storageName": null,
    "storageType": null,
    "tags": {
        "sentinelone": []
    },
    "threatRebootRequired": false,
    "totalMemory": 7926,
    "updatedAt": "2023-03-17T10:25:37.489066Z",
    "userActionsNeeded": [],
    "uuid": "be20d732fc87bd479e0a"
}
```

### Sample Query for Agents

> ```sql="Total Agents"

Reviewer: ```suggestion
```sql title="Total Agents"
```

Author: Resolved

The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.

## Viewing SentinelOne Dashboards​

* All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.

 You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.

* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.

> ### Agent Overview Dashboard

Author: Resolved

**SentinelOne - Agent Overview** provides valuable insights related to SentinelOne agents deployed across an organization. The dashboard includes widgets that display key information on the total number of agents, active threats, and infected agents. It also provides information on the distribution of agents by mitigation, machine type, and scan status.

Additionally, the dashboard includes information on the frequency of agents by installer type and the top 10 agents' operating systems and group summary. With this dashboard, security teams can easily monitor the performance of agents and ensure that they are adequately protecting their organization from potential threats.

The SentinelOne - Agent Overview enables organizations to quickly identify any issues with agent deployment or performance and take corrective action to improve their overall security posture. It is a powerful tool for any organization looking to ensure the effectiveness and efficiency of their threat protection solutions.
<br/><img src={useBaseUrl('img/integrations/saas-cloud/SentinelOne-Agent-Overview.png')} alt="SentinelOne-Agent-Overview.png" width="600"/>

> ### Threat Overview Dashboard

Reviewer: ```suggestion
### Threat Overview
```

Author: Resolved

**SentinelOne - Threat Overview** provides a comprehensive overview of threats detected by SentinelOne, a threat intelligence and response platform. The dashboard includes widgets that display key information on unresolved threats, suspicious threats detected, threats to be mitigated, resolved threats, and threats that require a reboot. The dashboard also provides insights into the types of threats detected, their current status, and their classification source. With this dashboard, security teams can quickly identify and respond to potential security threats, minimizing the risk of security breaches and data loss.
<br/><img src={useBaseUrl('img/integrations/saas-cloud/SentinelOne-Threat-Overview.png')} alt="SentinelOne-Threat-Overview.png" width="600"/>

> ### Threat Analysis Dashboard

Reviewer: ```suggestion
### Threat Analysis
```

Author: Resolved

**SentinelOne - Threat Analysis** provides a comprehensive view of the threat landscape and enables security teams to quickly identify and respond to potential threats. The dashboard includes widgets that display key information on the confidence level of threats detected, their location across various sites, the top initiated threats, and the engines used to detect them. The dashboard also provides insights into the status of threats over time, recent incidents, and the most infected devices.

With the information provided by this dashboard, security teams can effectively prioritize and manage their response to potential security threats. They can quickly identify the most critical threats, monitor the effectiveness of their threat detection engines, and respond to the most heavily infected devices. The "SentinelOne - Threat Analysis" is a valuable tool for any organization looking to proactively manage their cybersecurity risk.

> <br/><img src={useBaseUrl('img/integrations/saas-cloud/SentinelOne-Threat-Analysis.png')} alt="SentinelOne-Threat-Analysis.png" width="600"/>

Reviewer: ```suggestion

<img src={useBaseUrl('img/integrations/saas-cloud/SentinelOne-Threat-Analysis.png')} alt="SentinelOne-Threat-Analysis.png" width="600"/>
```

Author: Resolved

description: The Sumo Logic App for SentinelOne offers functionality for monitoring and analyzing your organization's SentinelOne Threats and Agents logs.
---

import useBaseUrl from '@docusaurus/useBaseUrl';

<img src={useBaseUrl('img/send-data/sentinelone-icon.png')} alt="sentinelone-icon.png" width="100" />

The SentinelOne App for Sumo Logic provides security professionals with a comprehensive view of their organization's security posture. The app, based on Sumo Logic’s SentinelOne Source, allows you to quickly ingest data from your SentinelOne agents into Sumo Logic for real-time analysis.

With the SentinelOne App, you can gain valuable insights into your endpoint security data, including threat intelligence, endpoint activity, and system health. You can monitor your endpoints for suspicious behavior and quickly respond to any security incidents that may arise. 

The app provides pre-built dashboards that enable you to quickly visualize your SentinelOne data and gain insights into your organization's security posture. You can customize these dashboards to meet your specific needs and monitor the metrics that matter most. With the SentinelOne App for Sumo Logic, you can:

> * Ingest data from SentinelOne agents into Sumo Logic for real-time analysis and response.

Reviewer 2: ```suggestion
* Ingest data from SentinelOne agents into Sumo Logic for real-time analysis and response
```

---
id: sentinelone
title: Sumo Logic App for SentinelOne
sidebar_label: SentinelOne
description: The Sumo Logic App for SentinelOne offers functionality for monitoring and analyzing your organization's SentinelOne Threats and Agents logs.
---

import useBaseUrl from '@docusaurus/useBaseUrl';

<img src={useBaseUrl('img/send-data/sentinelone-icon.png')} alt="sentinelone-icon.png" width="100" />

> The SentinelOne App for Sumo Logic is designed to provide security professionals with a comprehensive view of their organization's security posture. Based on Sumo Logic’s SentinelOne Source, this app allows you to quickly ingest data from your SentinelOne agents into Sumo Logic for real-time analysis.

Reviewer: ```suggestion
The SentinelOne App for Sumo Logic provides security professionals with a comprehensive view of their organization's security posture. The app, based on Sumo Logic’s SentinelOne Source, allows you to quickly ingest data from your SentinelOne agents into Sumo Logic for real-time analysis.
```

With the SentinelOne App, you can gain valuable insights into your endpoint security data, including threat intelligence, endpoint activity, and system health. You can monitor your endpoints for suspicious behavior and quickly respond to any security incidents that may arise. 

The app provides pre-built dashboards that enable you to quickly visualize your SentinelOne data and gain insights into your organization's security posture. You can customize these dashboards to meet your specific needs and monitor the metrics that matter most. With the SentinelOne App for Sumo Logic, you can:

> * Ingest data from SentinelOne agents into Sumo Logic for real-time analysis and response

Reviewer 3: ```suggestion
* Ingest data from SentinelOne agents into Sumo Logic for real-time analysis and response.
```

* Monitor your endpoints for suspicious behavior and respond to security incidents in real-time
* Gain valuable insights into your organization's security posture with pre-built dashboards
* Customize dashboards to meet your specific needs and monitor the metrics that matter most to you

In summary, the SentinelOne App for Sumo Logic provides security professionals with the tools they need to monitor and respond to security incidents in real-time, ensuring the protection of their organization's critical assets.

## Log Types

The Sumo Logic App for SentinelOne consumes Threats and Agents logs, refer to the [Threats](https://usea1-partners.sentinelone.net/api-doc/api-details?category=threats&api=get-threats) and [Agents](https://usea1-partners.sentinelone.net/api-doc/api-details?category=agents&api=get-agents) documentation.

> ### Sample Log Messages

Reviewer 3: It'd be great if we could write both log messages under the Sample Log Messages heading and queries under the Sample Queries section.

Reviewer: Agreed! @priyansh-sumo , I've just fixed this

}
```

### Sample Query for Threats

```sql title="Suspicious threats detected"
_sourceCategory=sentinelone  threatInfo
| json "id", "threatInfo.incidentStatus", "threatInfo.classificationSource", "threatInfo.confidenceLevel", "threatInfo.detectionEngines", "threatInfo.initiatedBy", "threatInfo.rebootRequired", "threatInfo.createdAt", "threatInfo.detectionType", "threatInfo.mitigationStatus", "threatInfo.analystVerdict", "threatInfo.threatName", "agentRealtimeInfo.siteName","agentRealtimeInfo.agentOsType", "agentRealtimeInfo.agentMachineType" as id, incidentStatus, classificationSource, confidenceLevel, detectionEngines, initiatedBy, rebootRequired, createdAt, detectionType, mitigationStatus, analystVerdict, threatName, siteName, agentOsType, agentMachineType nodrop
| where siteName matches "{{site}}" or isNull(siteName)
| where mitigationStatus matches "{{mitigation_status}}" or isNull(mitigationStatus)
| where agentOsType matches "{{os}}" or isNull(agentOsType)
| where agentMachineType matches "{{machine}}" or isNull(agentMachineType)
| where confidenceLevel matches "{{confidence_level}}" or isNull(confidenceLevel)
| where confidenceLevel = "suspicious"
| count_distinct(id)
```

> **Sample Agents Log Messages**

Reviewer: Don't need this (title is included in the snippet below)
```suggestion
```

 You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.

* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.

### Agent Overview

**SentinelOne - Agent Overview** provides valuable insights related to SentinelOne agents deployed across an organization. The dashboard includes widgets that display key information on the total number of agents, active threats, and infected agents. It also provides information on the distribution of agents by mitigation, machine type, and scan status.

Additionally, the dashboard includes information on the frequency of agents by installer type and the top 10 agents' operating systems and group summary. With this dashboard, security teams can easily monitor the performance of agents and ensure that they are adequately protecting their organization from potential threats.

> The SentinelOne - Agent Overview enables organizations to quickly identify any issues with agent deployment or performance and take corrective action to improve their overall security posture. It is a powerful tool for any organization looking to ensure the effectiveness and efficiency of their threat protection solutions.

Reviewer 3: ```suggestion
The SentinelOne Agent Overview enables organizations to quickly identify any issues with agent deployment or performance and take corrective action to improve their overall security posture. It is a powerful tool for any organization looking to ensure the effectiveness and efficiency of its threat protection solutions.
```
